### PR TITLE
Enhance store config errors by printing (println)

### PR DESCRIPTION
**Pull request by matthiasbeyer**

### CODE REVIEW

**Comment by neithernut** [comment]
You also did some reformatting here.

File: libimagstore/src/configuration.rs [hunk 1]
```diff
@@ -49,15 +49,23 @@ pub fn config_is_valid(config: &Option<Value>) -> bool {
 
     fn has_key_with_string_ary(v: &BTreeMap<String, Value>, key: &str) -> bool {
         v.get(key)
-            .map(|t| match t {
-                &Value::Array(ref a) => a.iter().all(|elem| {
-                    match elem {
-                        &Value::String(_) => true,
-                        _ => false,
+            .map(|t| {
```
Comment (matthiasbeyer) [inline libimagstore/src/configuration.rs]: @neithernut I assume you mean this new scope here with "reformatting"?

Comment (neithernut) [inline libimagstore/src/configuration.rs]: No, you added a bunch of empty lines in another place.

**Comment by matthiasbeyer** [comment]
Removed the newlines you mentioned.

File: libimagstore/src/configuration.rs (hunk 1 above)
Comment (neithernut) [inline libimagstore/src/configuration.rs]: Now that you said it: why did you add the extra scope here?

Comment (matthiasbeyer) [inline libimagstore/src/configuration.rs]: I did some `println!()` foo to debug. Will remove.

**Comment by matthiasbeyer** [comment]
All of these `println!()` should be `write!(stderr(), ...`s.
